inst/rmarkdown/templates/quantitative_research_in_psych_prereg/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,411 @@
+
+Enter your response here.
+
+
+## AP6
+### Inference criteria
+<!-- Specify the criteria used for inferences (e.g., p values, Bayes factors, effect size measures) and the thresholds for accepting or rejecting your hypotheses. If possible, define a smallest effect size of interest. If inference criteria differ between hypotheses, specify separately for each hypothesis and respective statistical model by explicitly referring to the numbers of the hypotheses. Describe which effect size measures will be reported and how they are calculated. -->
+
+Enter your response here.
+
+
+## AP7
+### Exploratory analysis (optional)
+<!-- Describe any exploratory analyses to be conducted with your data. Include here any planned analyses that are not confirmatory in the sense of being a direct test of one of the specified hypotheses. -->
+
+Enter your response here.
+
+
+## AP8
+### Other information (optional)
+
+Enter your response here.
+
+
+
+# Other
+<!-- Other information, optional
+NOTE: If needed, multiple lines with other information can be included -->
+
+## O1
+### Other information (optional)
+<!-- If there is any additional information that you feel needs to be included in your preregistration, please enter it here. Literature cited, disclosures of any related work such as replications or work that uses the same data, or other context that will be helpful for future readers would be appropriate here. -->
+
+Enter your response here.
+
+
+
+<!-- References of the template	
+
+Aczel, B., Szaszi, B., Sarafoglou, A., … Wagenmakers, E.-J. (2020). A consensus-based transparency checklist. Nature Human Behaviour, 4(1), 4–6. https://doi.org/10.1038/s41562-019-0772-6	
+
+American Psychological Association. (2020). Publication manual of the American Psychological Association (7th ed.). https://doi.org/10.1037/0000165-000	
+
+Appelbaum, M., Cooper, H., Kline, R. B., Mayo-Wilson, E., Nezu, A. M., & Rao, S. M. (2018). Journal article reporting standards for quantitative research in psychology: The APA Publications and Communications Board task force report. American Psychologist, 73(1), 3–25. https://doi.org/10.1037/amp0000191	
+
+Bowman, S. D., DeHaven, A. C., Errington, T. M., Hardwicke, T. E., Mellor, D. T., Nosek, B. A., & Soderberg, C. K. (2016). OSF Prereg Template. Retrieved from osf.io/preprints/metaarxiv/epgjd	
+
+Simonsohn, U., Simmons, J., & Nelson, L. (2017). AsPredicted. Retrieved from https://aspredicted.org/messages/terms.php 1/15/2020	
+
+Van den Akker, O., Weston, S. J., Campbell, L., Chopik, W. J., Damian, R. I., Davis-Kean, P., Hall, A., Kosie, J., Kruse, E., Olsen, J., Stuart, R., Valentine, K., van 't Veer, A., & Bakker, M. (2019, November 20). Preregistration of secondary data analysis: A template and tutorial. https://doi.org/10.31234/osf.io/hvfmr
+-->
+
+\newpage
+# References
+## 
```

Comment:
Did you add this empty level 2-heading here intentionally?

Reply:
I just checked this - this empty level 2-heading is part of all other prereg-templates too! So I would suggest to just leave it there?

Reply:
Ah, yes. Sorry.